src/components/AddItem.jsx

```diff
@@ -0,0 +1,170 @@
+import { useEffect, useState } from 'react';
+import {
+	Box,
+	Drawer,
+	DrawerBody,
+	DrawerHeader,
+	DrawerOverlay,
+	DrawerContent,
+	DrawerCloseButton,
+	FormControl,
+	FormLabel,
+	RadioGroup,
+	Radio,
+	Input,
+	Button,
+	useToast,
+	Flex,
+	Stack,
+	Heading,
+	Text,
+} from '@chakra-ui/react';
+import { addItem } from '../api/firebase';
+
+export function AddItem({
+	data,
+	listId,
+	isOpen,
+	onClose,
+	btnRef,
+	initialValue,
+}) {
+	const [timeframe, setTimeframe] = useState('7');
+	const [itemName, setItemName] = useState(initialValue);
+	const toast = useToast();
+
+	const onItemChange = (e) => setItemName(e.target.value);
+
+	// function to validate itemName input that returns error messages if any - called onFormSubmit
+	function validateItemInput(data, trimmedItemName) {
+		// 1- checks for empty inputs
+
+		if (!trimmedItemName) {
+			toast({
+				title: 'No item name entered',
```

Comment:
@katherineyuneman Same here, do you think having a title is necessary? If so, we should add one to list item page. I feel like takes more space and is basically a repeat of the description.